packages/components/molecules/f-skeleton-loader/src/components/skeletons/CarouselCardBackground.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class="c-skeleton-loader-carouselCardBackground c-skeleton-loader-bone">
+        <image-block />
+    </div>
+</template>
+
+<script>
+import ImageBlock from './Image.vue';
+
+export default {
+    name: 'CarouselCardBackground',
+    components: { ImageBlock }
+};
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Do you think it's worth using a mix of `scoped` and `module` in these components? i.e. `scoped` when there are styles targeting child elements (I assume that's what the `/deep/` selectors are doing?) and `module` when not?

Reply:
Potentially with some investigation work that could be feasible. However, we don't necessarily know when a parent or new component might need to target and change a child. Mixing could make it harder to understand and manage. Yes, the /deep/ allows child access more than one level deep.

Reply:
Sure, makes sense - best to go with one and stick with it in the same context 👍🏻